formulas/wick-base/functions/wick-make-user

```diff
@@ -21,6 +21,10 @@
 # If the given user already exists and no options are given `usermod` will not
 # be run.
 #
+# When `--groups` is specified the list of groups specified is added to the
+# existing group memberships. The user's initial group is still automatically
+# created. See useradd or usermod for further information.
```

Comment:
Both `useradd` and `usermod` should have backticks. They aren't English words and are instead code.